npm run test fails

$ npm run test:run

```
> typings-for-css-modules-loader@1.0.0 test:run /home/davidm/src/Jimdo/typings-for-css-modules-loader
> babel-node ./node_modules/webpack/bin/webpack --config ./test/webpack.config.babel.js && npm run test:diff

ts-loader: Using typescript@1.8.10 and /home/davidm/src/Jimdo/typings-for-css-modules-loader/test/tsconfig.json
Hash: 743acee40bc0d406534c
Version: webpack 1.14.0
Time: 1626ms
    Asset     Size  Chunks             Chunk Names
bundle.js  3.08 kB       0  [emitted]  main
   [1] ./test/example.css 0 bytes [built] [failed]
   [2] ./test/example-camelcase.css 0 bytes [built] [failed]
   [3] ./test/example-namedexport.css 0 bytes [built] [failed]
   [4] ./test/example-camelcase-namedexport.css 0 bytes [built] [failed]
    + 1 hidden modules

ERROR in ./test/entry.ts
(1,36): error TS2307: Cannot find module './example.css'.
```

--- a/package.json
+++ b/package.json
@@ -32,18 +32,21 @@
   },
   "devDependencies": {
     "babel-cli": "6.10.1",
+    "babel-core": "~6.21.0",
     "babel-eslint": "6.1.0",
-    "babel-loader": "^6.2.5",
-    "babel-polyfill": "^6.13.0",
+    "babel-loader": "~6.2.5",
+    "babel-polyfill": "~6.13.0",
     "babel-preset-es2015": "6.9.0",
     "babel-preset-stage-0": "6.5.0",
+    "css-loader": "~0.26.1",
     "eslint": "2.13.1",
     "eslint-plugin-babel": "3.3.0",
-    "ts-loader": "^0.8.2",
-    "typescript": "^1.8.10",
-    "webpack": "^1.13.2"
+    "ts-loader": "~0.8.2",
+    "typescript": "~1.8.10",
+    "webpack": "~1.13.2"
   },
   "peerDependencies": {
+    "babel-core": "^6.0.0",
     "css-loader": ">=0.23.1"
   },
   "repository": {

